skin.json

```diff
@@ -1,13 +1,58 @@
 {
+    "Configuration": [
+        {
+            "Name": "Wallet Visibility",
+            "ToolTip": "Hide wallet balance",
+            "Type": "CheckBox",
+            "Value": false
+        },
+        {
+            "Name": "Name Visibility",
+            "ToolTip": "Hide user name from top right corner",
+            "Type": "CheckBox",
+            "Value": false
+        }
+    ],
 	"Patches": [
         {
             "MatchRegexString": "^Steam$",
-            "TargetCss": "Steam SpaceTheme/main.css"
+            "TargetCss": "Steam SpaceTheme/main.css",
+            "Statement": [
+                {
+                    "If": "Wallet Visibility",
+                    "Equals": true,
+                    "True": {
+                        "TargetCss": "Steam SpaceTheme/files/theme/config/WalletVisibility.css"
+                    },
+                    "False": {}
+                },
+                {
+                    "If": "Name Visibility",
+                    "Equals": true,
+                    "True": {
+                        "TargetCss":"Steam SpaceTheme/files/theme/config/NameVisibility.css"
+                    },
+                    "False": {}
+                }
+            ]
         },
         {
             "MatchRegexString": "Friends List",
             "TargetJs": "friends.custom.js",
             "TargetCss": "friends.custom.css"
+        },
+        {
+            "MatchRegexString": "^Account",
```

Comment:
and here i don't know which side this is applied to (maybe just tell me where it will be patched)